Source/UI/Packages/Installing/View Controllers/ProgressController.mm

```diff
@@ -64,6 +64,18 @@ - (void) viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
 }
 
+- (void) justClose
```

Comment:
Can we get a more descriptive name here like `closeWithoutAnyPostInstallActions`?